espflash/src/connection.rs

```diff
@@ -119,43 +125,77 @@ impl Connection {
         Ok(reset_after_flash(&mut self.serial, pid)?)
     }
 
-    // Reset the device to flash mode
-    pub fn reset_to_flash(&mut self, extra_delay: bool) -> Result<(), Error> {
-        if self.port_info.pid == USB_SERIAL_JTAG_PID {
-            self.serial.write_data_terminal_ready(false)?;
-            self.serial.write_request_to_send(false)?;
+    #[cfg(unix)]
```

Comment:
Why is `classic_reset` only available for *nix systems?

Reply:
That's my bad, I got it the wrong way around. I'll fix that.